### PR TITLE
[FIX] website: prevent snippet drop in product_catalog

### PR DESCRIPTION
**Pull request by detrouxdev**
Before this commit, a user would be able to drop pretty much any
snippet inside the product_catalog (pricelist) snippet.
This was not ideal as the snippet wasn't designed to receive
any other snippets inside of it.

With this commit, the configuration of the snippet is closer
to snippets such as timeline, allowing lines to be moved around
without allowing snippets to be dropped inside it.

task-2655171

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Review by bvr-odoo — CHANGES_REQUESTED** [review]
only two things:
- It's a fix and not an imp
- check in which version the bug appears, test it and retarget your PR on it.

**Review by bvr-odoo — APPROVED** [review]

File: addons/website/views/snippets/s_product_catalog.xml [hunk 1]
```diff
@@ -70,11 +70,10 @@
                 <!-- Note: no need of extra dependency thanks to the apply-to -->
             </t>
         </div>
-        <div data-selector=".s_product_catalog_dish"/>
+        <div data-selector=".s_product_catalog_dish" data-drop-near=".s_product_catalog_dish"/>
     </xpath>
-    <xpath expr="//div[@data-js='SnippetMove']" position="attributes">
+    <xpath expr="//div[@data-js='SnippetMove'][contains(@data-selector,'section')]" position="attributes">
```
Comment (qsm-odoo) [inline addons/website/views/snippets/s_product_catalog.xml:75]: Missing space after the `,` but in fact I don't think that change is needed at all?

**Review by qsm-odoo — CHANGES_REQUESTED** [review]

File: addons/website/views/snippets/s_product_catalog.xml (hunk 1 above)
Comment (bvr-odoo) [reply addons/website/views/snippets/s_product_catalog.xml:75]: without this fix, any snippet can be dropped into the product catalog snippet...
IMO we dont want it

Comment (bvr-odoo) [reply addons/website/views/snippets/s_product_catalog.xml:75]: oh you mean just this line

Comment (detrouxdev) [reply addons/website/views/snippets/s_product_catalog.xml]: It's my bad for copy pasting from timeline without actually understanding what this was used for. It was indeed useless so I removed it. Sorry 😅

**Review by qsm-odoo — APPROVED** [review]
@robodoo r+

**Pull request closed without merging**